config/locales/docs/sk.yml

```diff
@@ -830,7 +830,9 @@ sk:
             organization_id:
               description: Identifikátor právnickej osoby
             stakeholder_type_id:
-              description: Identifikátor typu zainteresovanej osoby
+              description: Identifikátor typu štatutárneho orgánu
```

Comment:
preco statutarneho ked je to `stakeholder_type`?

Reply:
lebo je to ten omyl a bude sa to vymazavat, je to tam len docasne

Reply:
aha ok